src/styles/index.scss

```
@@ -1,3 +1,119 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  background-color: grey;
+}
+
+p {
+  margin: 0;
+}
+
+.first-block {
+  width: 200px;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.par {
+  font-size: 12px;
+  color: #060B35;
+  font-weight: 500;
+  line-height: 18px;
+  font-style: normal;
+  font-family: Roboto, sans-serif;
+  margin-top: 40px;
+  text-align: justify;
+  margin-left: 16px;
+}
+
+img {
+  width: 160px;
+  height: 134px;
+  margin-top: 32px;
+  margin-left: 19px;
+  margin-right: 19px;
+}
+
+.par2 {
+  margin-left: 16px;
+  color: #616070;
+  font-size: 10px;
+  font-weight: 400;
+  margin-top: 4px;
+  margin-bottom: 16px;
+}
+
+.par3 {
+  color: #060B35;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  margin-left: auto;
+  margin-right: 16px;
+  display: flex;
+  align-items: center;
+}
+
+.par4 {
+  color: #616070;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  margin-left: 16px;
+  margin-top: 24px;
+}
+
+.par5 {
+  color: #060B35;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  margin-left: 100px;
+  margin-top: 24px;
+}
+
+.stars {
+  display: flex;
+  margin-left: 16px;
+}
+
+.button {
+  display: flex;
+  height: 40px;
+  color: #fff;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  border-radius: 5px;
+  background: #00acdc;
+  text-align: center;
+  align-items: center;
+  justify-content: center;
+  margin: 58px 16px 16px;
+}
+
+.review-section {
+  display: flex;
+}
+
+.price {
+  display: flex;
+}
+
+.button:hover {
+  border-radius: 5px;
+  border: 1px solid #00acdc;
+  background: #fff;
+  color:  #00acdc;
+  text-align: center;
+  font-family: Roboto, sans-serif;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 16px; /* 114.286% */
```

```suggestion
  line-height: 16px;
```

Remove all comments from the code